src/main/java/biz/netcentric/maven/extension/repofromenv/CompositeMap.java

```diff
@@ -0,0 +1,111 @@
+package biz.netcentric.maven.extension.repofromenv;
```

Comment:
Could we just use https://commons.apache.org/proper/commons-collections/apidocs/org/apache/commons/collections4/map/CompositeMap.html for this? (it's adding a dependency, but that wouldn't hurt much?)

Reply:
yes, wasn't aware of it. Will use it instead.